Makefile

```diff
@@ -187,7 +188,14 @@ pull:
 	git pull
 	$(MAKE) submodule-sync
 
-.PHONY: submodule pull submodule-sync
+# Pull from git master. To be used in CI or by developers, assumes the
+# remote is called 'origin' (git default). Will fail on conflicts
+# Note: in a branch, this will get the latest from master. In master it has no effect.
+# (pull == fetch + merge )
+git.pullmaster:
+	git merge master
```

Comment:
do you need a git fetch then now ? also maybe update the comment `(pull == fetch + merge )` part